### PR TITLE
Discover search & tests

### PR DESCRIPTION
**Pull request by lukemeierj**
* Now searching text in discover searches _all_ activities
* Add tests

To test:
* Favorite one topic
* Search the title of a activity that doesn't have that topic
* If it shows up, you're g2g

### CODE REVIEW

**Comment by lukemeierj** [comment]
Hell, baby.

**Comment by noahtallen** [comment]
HOT

File: BuddiesUITests/TestReport.swift [hunk 1]
```diff
@@ -1,29 +0,0 @@
-//
```
Comment (noahtallen) [inline BuddiesUITests/TestReport.swift:1]: this file supposed to be deleted?

**Review by noahtallen — APPROVED** [review]
the change works and I like the coverage!

**Pull request merged**